controllers/allProducts.js

```diff
@@ -0,0 +1,15 @@
+import productServer from '../models/productServer';
+
+export default {
+    getAllProducts:  function () {
```

Comment:
A controller is a specific type of middleware that accepts request and response object. It's purpose is to take all the request data needed to prepare the response data and send it back to the client.
Taking this, the routes/api.js keeps too much responsibilities, which should be delegated to the specific controllers instead.